app/src/main/java/fr/free/nrw/commons/contributions/WikipedaiInstructionsDialogFragment.kt

```diff
@@ -0,0 +1,72 @@
+package fr.free.nrw.commons.contributions
+
+import android.net.Uri
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.view.WindowManager
+import androidx.fragment.app.DialogFragment
+import fr.free.nrw.commons.R
+import fr.free.nrw.commons.Utils
+import fr.free.nrw.commons.upload.SimilarImageDialogFragment
+import kotlinx.android.synthetic.main.dialog_add_to_wikipedia_instructions.*
+import org.wikipedia.dataclient.WikiSite
+import javax.inject.Inject
+
+/**
+ * Dialog fragment for displaying instructions for editing wikipedia
+ */
+class WikipedaiInstructionsDialogFragment : DialogFragment() {
```

Comment:
A typo in the class name:

```suggestion
class WikipediaInstructionsDialogFragment : DialogFragment() {
```